gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -113,6 +114,64 @@ func TestCallMultiple_Integration(t *testing.T) {
 	assert.Equal(t, expected, got)
 }
 
+func TestCallMultipleEvent_Integration(t *testing.T) {
+	// Set up in-memory node
+	config, _ := integration.TestingNodeConfig(t, gnoenv.RootDir())
+	node, remoteAddr := integration.TestingInMemoryNode(t, log.NewNoopLogger(), config)
+	defer node.Stop()
+
+	// Init Signer & RPCClient
+	signer := newInMemorySigner(t, "tendermint_test")
+	rpcClient, err := rpcclient.NewHTTPClient(remoteAddr)
+	require.NoError(t, err)
+
+	// Setup Client
+	client := Client{
+		Signer:    signer,
+		RPCClient: rpcClient,
+	}
+
+	// Make Tx config
+	baseCfg := BaseTxCfg{
+		GasFee:         "10000ugnot",
+		GasWanted:      8000000,
+		AccountNumber:  0,
+		SequenceNumber: 0,
+		Memo:           "",
+	}
+
+	// Make Msg configs
+	msg1 := MsgCall{
+		PkgPath:  "gno.land/r/demo/event",
+		FuncName: "Emit",
+		Args:     []string{"first_value"},
+		Send:     "",
+	}
+
+	// Same call, different argument
+	msg2 := MsgCall{
+		PkgPath:  "gno.land/r/demo/event",
+		FuncName: "Emit",
+		Args:     []string{"second_value"},
+		Send:     "",
+	}
+
+	// Execute call
+	res, err := client.Call(baseCfg, msg1, msg2)
+	assert.Nil(t, err)
+
+	// check response from block_results to get event
+	blockRes, err := client.BlockResult(res.Height)
+	assert.Nil(t, err)
```

Comment:
If I'm not mistaken, there is an assert.NoError function that provides more details if the error is not nil.

Reply:
4be002898a9e9ee9f054736818b7b838ceb3ebd3